content/docs/command-reference/import.md

```diff
@@ -27,20 +27,25 @@ target file or directory (found at `path` in `url`), and tracks it in the local
 project. This makes it possible to update the import later, if the data source
 has changed (see `dvc update`).
 
-> Note that `dvc get` corresponds to the first step this command performs (just
-> download the data).
-
 > See `dvc list` for a way to browse repository contents to find files or
 > directories to import.
 
+> Note that `dvc get` corresponds to the first step this command performs (just
+> download the data).
+
 The imported data is <abbr>cached</abbr>, and linked (or copied) to the current
 working directory with its original file name e.g. `data.txt` (or to a location
 provided with `--out`). An _import `.dvc` file_ is created in the same location
 e.g. `data.txt.dvc` – similar to using `dvc add` after downloading the data.
 
-(ℹ️) DVC won't push or pull imported data to/from
-[remote storage](/doc/command-reference/remote), it will rely on it's original
-source.
+💡 Using an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+lets you transfer an import there (and link it in the <abbr>workspace</abbr>),
+without using the local file system.
```

Comment:
Note that I realized the to-cache strategy also applies to regular `import` (I think), so I added this note.

Reply:
Done. Please note iterative/dvc/issues/5623 when you have a change. Thanks